tests/forms_tests/widget_tests/test_select.py

```diff
@@ -119,37 +118,37 @@ def test_compare_to_str(self):
             self.widget(choices=[("1", "1"), ("2", "2"), ("3", "3")]),
             "num",
             2,
-            html=(
-                """<select name="num">
-                <option value="1">1</option>
-                <option value="2" selected>2</option>
-                <option value="3">3</option>
-                </select>"""
-            ),
+            html="""
+            <select name="num">
+              <option value="1">1</option>
+              <option value="2" selected>2</option>
+              <option value="3">3</option>
+            </select>
+            """,
         )
         self.check_html(
             self.widget(choices=[(1, 1), (2, 2), (3, 3)]),
             "num",
             "2",
-            html=(
-                """<select name="num">
-                <option value="1">1</option>
-                <option value="2" selected>2</option>
-                <option value="3">3</option>
-                </select>"""
-            ),
+            html="""
+            <select name="num">
+              <option value="1">1</option>
+              <option value="2" selected>2</option>
+              <option value="3">3</option>
+            </select>
+            """,
         )
         self.check_html(
             self.widget(choices=[(1, 1), (2, 2), (3, 3)]),
             "num",
             2,
-            html=(
-                """<select name="num">
-                <option value="1">1</option>
-                <option value="2" selected>2</option>
-                <option value="3">3</option>
-                </select>"""
-            ),
+            html="""
+            <select name="num">
+              <option value="1">1</option>
+              <option value="2" selected>2</option>
+              <option value="3">3</option>
+            </select>
+            """,
```

Comment:
I decided to remove parenthesis because there was some inconsistency between `test_select` and `test_radioselect`, some test had them, some didn't. What do you think, all test should have them or leave it as it is?

Reply:
I had to revert the indentation changes for `test_select` since we aim at not changing/reformatting code just for the sake of consistency.